app/views.py

```diff
@@ -326,13 +421,15 @@ def ip_list_sort(ret):
 def favicon():
     return send_from_directory(os.path.join(app.root_path, 'static'), 'favicon.ico', mimetype='image/vnd.microsoft.icon')
 
+
```

Comment:
Is double whitespace after a function the standard for the lint?

Reply:
two blank lines *before* a function is, yes. 🙂